Arf.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+enum
+{
+   SizeOfArray = 5,
+};
+
+static void ShowArray(const double array[], int sizeOfArray)
+{
+   for(int index = 0; index < sizeOfArray; index++)
+   {
+      printf("%8.3f", array[index]);
+      putchar('\n');
+   }
+}
+
+static void MultiplyElementsOfArrayByFactor(double array[], int sizeOfArray, double factor)
+{
+   for(int index = 0; index < sizeOfArray; index++)
+   {
+      array[index] *= factor;
+   }
+}
+
+int main(void)
+
+{
+   double array[SizeOfArray] = { 20.0, 17.66, 8.2, 15.3, 22.22 };
```

Comment:
```suggestion
   double array[] = { 20.0, 17.66, 8.2, 15.3, 22.22 };
```

The array size is set automatically if you initialize it like this